boards/xtensa/intel_adsp_ace15_mtpm/intel_adsp_ace15_mtpm_defconfig

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Looks like this got added by a script or something?  The kconfig tool uses Bourne shell comments ("#" lines) and doesn't invoke the C preprocessor.  These lines get flagged with a "malformed" warning by kconfig which then aborts the build.